### ISSUE TITLE
Remove all 'subprocess' package incidences

### ISSUE THREAD

**Issue by schmelz21** [cisagov/pe-reports#23]
# 🐛 Summary #

Python package subprocess is flagged by `bandit` check as "blacklisted".

## To reproduce ##

1. Run branch pre-commit: https://github.com/cisagov/pe-reports/tree/AL-working-v1.0

## Expected behavior ##

A `pre-commit run --all-files` should render no `bandit` errors.

## Any helpful log output or screenshots ##

Paste the results here:

```
>> Issue: [B404:blacklist] Consider possible security implications associated with subprocess module.
   Severity: Low   Confidence: High
   Location: src/pe_reports/report_generator.py:29
   More Info: https://bandit.readthedocs.io/en/latest/blacklists/blacklist_imports.html#b404-import-subprocess

29      import subprocess
```

**Comment by cduhn17** [comment]
Adding this to inform about being flagged for 'subprocess' usage [https://github.com/PyCQA/bandit/issues/455](https://github.com/PyCQA/bandit/issues/455) and [https://github.com/PyCQA/bandit/issues/333#issuecomment-404103697](https://github.com/PyCQA/bandit/issues/333#issuecomment-404103697)

**Comment by cduhn17** [comment]
This issue was merged into develop branch per PR#31
